docs/source/quickstart.rst

```diff
@@ -75,7 +88,7 @@ Define and Debug Services
 -------------------------
 
 Services are the core components of BentoML, where the serving logic is defined. With the model
-saved in the model store, we can define the :ref:`service <service-definition-page>` by creating a
+saved in the model store, we can define the :ref:`web service <service-definition-page>` by creating a
```

Comment:
```suggestion
saved in the model store, we can define the :ref:`service <service-definition-page>` by creating a
```

Just "service"? since it's called `bentoml.Service` in the API?

Reply:
ah right. I guess I was trying to clarify that what you're getting is a web service. There's no other types of services that you can instantiate right?